src/pages/UserSettings/Teams.vue

```diff
@@ -0,0 +1,171 @@
+<script>
+import { mapActions, mapGetters } from 'vuex'
+import { clearCache } from '@/vue-apollo'
+
+import ManagementLayout from '@/layouts/ManagementLayout.vue'
+import ConfirmDialog from '@/components/ConfirmDialog'
+
+export default {
+  components: {
+    ManagementLayout,
+    ConfirmDialog
+  },
+  data() {
+    return {
+      dialogRemoveUser: false,
+      headers: [
+        { text: 'Name', value: 'name' },
+        { text: 'URL', value: 'slug' },
+        { text: 'Role', value: 'role' },
+        { text: 'Actions', value: 'id' }
+      ],
+      isRemovingUser: false,
+      removeTenant: null
+    }
+  },
+  computed: {
+    ...mapGetters('user', ['memberships']),
+    ...mapGetters('tenant', ['tenant', 'tenants'])
+  },
+  methods: {
+    ...mapActions('tenant', ['getTenants', 'setCurrentTenant']),
+    role(item) {
+      let role
+      switch (item) {
+        case 'USER':
+          role = 'User'
+          break
+        case 'READ_ONLY_USER':
+          role = 'Restricted User'
+          break
+        case 'TENANT_ADMIN':
+          role = 'Administrator'
+          break
+        default:
+          role = ''
+          break
+      }
+      return role
+    },
+    async removeUser(removalTenant) {
+      this.isRemovingUser = true
+      const initialTenant = this.tenant
+
+      let membershipsObject = {}
+      for (const membership of this.memberships) {
+        membershipsObject[membership.tenant.id] = membership.id
+      }
+      const membershipId = membershipsObject[removalTenant.id]
```

Comment:
there's probably a better way to do this? but I wanted to get just the relevant information for identifying and switching tenants into one spot